labelCloud/model/point_cloud.py

```diff
@@ -210,6 +226,30 @@ def label_counts(self) -> Optional[Dict[str, int]]:
             return counter
         return None
 
+    @property
+    def has_label(self) -> bool:
+        return self.labels is not None
+
+    def update_colors_selected_points(
```

Comment:
Can you add a doc string here that explains what the function does and how it does it?

Reply:
done here [45e06f0](https://github.com/ch-sa/labelCloud/pull/104/commits/45e06f0addf2f3ee3eb5b63427933e118424c6dc)